### ISSUE TITLE
Failed to open .bcp file from WK 1.3

### ISSUE THREAD

**Issue by black10500** [AdrienTD/wktools#4]
Hello, I have a problem opening a .bcp file. 

I'm trying to open a .bcp file from WK version 1.3 (CD), because there are sound files with local translation (Polish language) which I would like to transfer to version 1.4. Unfortunately when I try to open the file the bcpview program displays the error "Failed to allocate memory for the position/size table".

![bcpviewer_error](https://github.com/AdrienTD/wktools/assets/150305814/df62e675-8c7c-41d8-b754-937d77b15d38)

Is there a chance to bypass this problem? At least to export the voice files? :)

**Comment by AdrienTD** [comment]
Hi!

Be sure that you are using the latest version (v1.0.1) of BCP Viewer, as it might fix this particular bug. A build can be found here:
https://cdn.discordapp.com/attachments/689898396323938342/694855044989714522/BCP_Viewer.zip

Let me know if this issue is still present in the latest version.

**Comment by black10500** [comment]
Yes, it works!

This new version of BCP Viewer opens a .bcp file from WK version 1.3. I already exported the file and following your [instructions](https://steamcommunity.com/groups/WARKC/discussions/0/366298942097887742/) I created a patch that adds the Polish language in version a1.4 (with your WKUUP patch).

So far it works - both text and audio :D
![PL_WK14_01](https://github.com/AdrienTD/wktools/assets/150305814/913e46b4-7a79-418a-a476-01c59f5bb6d2)

Thank you for your help, it's really amazing that still someone is taking care of this game. Maybe I should publish the language pack? So far I haven't found a Polish version of 1.4 anywhere. Maybe someone will be happy :)
